priprava/jazyky-nastroje/instalace-python.md

```diff
@@ -0,0 +1,21 @@
+## Instalace jazyka Python
+
+### Windows
+Pokud máte počítač s Windows, následujte tyto kroky:
+
+1. Ujistěte se, že máte aktualizovaný systém. To zajistíte nejjednodušeji tak, že svůj počítač restartujete. 
+1. Stáhněte si instalaci jazyka Python z [tohoto odkazu](https://www.python.org/ftp/python/3.9.10/python-3.9.10-amd64.exe). Stažený soubor spusťte. Rozběhne se průvodce instalací.
+1. Na úvodní obrazovce je **velmi důležité** zaškrtnout volbu **Add Python 3.9 to PATH** (viz obrázek)
+
+    ::fig[Instalace Pythonu]{src=assets/python-setup.png}
+    
+1. Klikněte na **Install Now**, odsouhlaste případné otázky ohledně změn na vašem počítači a vyčkejte dokončení instalace. Jakmile instalace skončí, zavřete okno tlačítkem **Close**.
+
+### Mac OS
+Pokud máte počítač s Mac OS, je velmi pravděpodobné, že už máte Python nainstalovaný. Ověřte si instalaci (viz sekce [Ověření instalace](./overeni.md)).
```

Comment:
Jak se odkazovat na kapitoly? Bude `[Ověření instalace](./overeni.md)` fungovat?

Reply:
Dobrá otázka. Ale hrajem přece šifrovačku a občas `stage` shodime, ne?

Reply:
Ok, prostě to vyzkoušíme 🙂